Add clarification between self-installed and cloud instances of Ubuntu 20 and 22 (#2245)
uyuni-project/uyuni-docs: modules/client-configuration/pages/clients-ubuntu.adoc

To be able to onboard using a regular user, you need to edit the [filename]``sudoers`` file.

[NOTE]
====
This issue happens with self-installed versions of {ubuntu}.
If the default user has been granted administrative privileges during installation time, a password is requiered to perform privilege escalation using [command]``sudo``.

> With cloud instances this does not happen since [command]``cloud-init`` automatically creates a file under [path]``/etc/sudoers.d`` and grants privilege escalation through [command]``sudo`` without the need for a password.

Reviewer: ```suggestion
With cloud instances this does not happen because [command]``cloud-init`` automatically creates a file under [path]``/etc/sudoers.d`` and grants privilege escalation through [command]``sudo`` without the need for a password.
```
We try to avoid "since" when because can be used.

Author: Wording changed @keichwa !

To be able to onboard using a regular user, you need to edit the [filename]``sudoers`` file.

[NOTE]
====

> This issue happens with self-installed versions of {ubuntu} since if the default user has been granted with administrative privileges

Author: Addressed! :smile:

> during installation time, it will requiere a password to perform privilege escalation using [command]``sudo``.

Author: Addressed! :smile:

> With cloud instances this normally doesn't happen since [command]``cloud-init`` creates automatically a file under [path]``/etc/sudoers.d`` 

Author: Addressed! :smile:

> which grants privilege escalation through [command]``sudo`` without the need of a password.

Reviewer 2: ```suggestion
and grants privilege escalation through [command]``sudo`` without the need for a password.
```
